src/tess_sip/tess_sip.py

```diff
@@ -80,6 +83,10 @@ def SIP(
         When True, will run SFF detrending.
     sff_kwargs : dict
         Dictionary of SFF key words to pass. See lightkurve's SFFCorrector.
+    custom_periods : None or numpy.ndarray
```

Comment:
I think this is called something else in `astropy.lombscargle`, let's use the same naming convention

Reply:
The `astropy.lombscargle` [documentation](https://docs.astropy.org/en/stable/api/astropy.timeseries.LombScargle.html#astropy.timeseries.LombScargle.power) just uses `frequency`, but I changed the variable name to `given_periods` and slightly updated the description to be more consistent with their language.